app/javascript/controllers/deposit_button_controller.js

```diff
@@ -1,13 +1,14 @@
 import { Controller } from "stimulus"
 
 export default class extends Controller {
-  static targets = ["depositButton", "globusRadioButton", "globusCheckbox", ]
+  static targets = ["depositButton", "globusRadioButton", "globusCheckbox", "globusMessage", ]
 
   connect() {
     this.updateDepositButtonStatus()
   }
 
   updateDepositButtonStatus(_event) {
+    this.globusMessageTarget.hidden = !this.globusRadioButtonTarget.checked
```

Comment:
Removed in https://github.com/sul-dlss/happy-heron/pull/2913/files#diff-33148bcaafd73019c39a8ffb129631d1f60488cbb143812810a9b9f3d21e90e3L12 

@justinlittman Can you confirm this was inadvertent?

Reply:
I'll assume so for now. Holler if not!